system/jlib/jmisc.cpp

```diff
@@ -1082,3 +1082,33 @@ std::pair<std::string, unsigned> getDafileServiceFromConfig(const char *applicat
         throw makeStringExceptionV(-1, "dafilesrv '%s': external service port not defined", dafilesrvName.str());
     return externalService;
 }
+
+#if defined(__linux__)
```

Comment:
minor: This would be a cleaner if it was testing a HAS_CONFIGURATION_HOOK symbol - just in case it gets implemented in windows at a future date.  Or is it because setenv is not supported?  Not worth changing for this PR though.

Reply:
ok, I will investigate windows support for this (it should be done so standalone tests that run in windows support these env. variables, but yes, as a separate issue/PR.